mesheryctl/internal/cli/root/system/update.go

```diff
@@ -41,11 +38,6 @@ var updateCmd = &cobra.Command{
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 
-		if _, err := os.Stat(utils.DockerComposeFile); os.IsNotExist(err) {
-			if err := utils.DownloadFile(utils.DockerComposeFile, fileURL); err != nil {
-				return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to download %s file from %s", utils.DockerComposeFile, fileURL)))
-			}
-		}
 		err := resetMesheryConfig()
 		if err != nil {
```

Comment:
`resetMesheryConfig` already handles downloading the docker-compose file
removed redundant code

TODO: fixing the update command to abort update when reset is aborted. Currently `resetMesheryConfig` does not return anything that can signal reset aborted.

Reply:
When running reset() from the update command, we can maybe pass in an argument that would suppress the confirmation question. If we still need to confirm with the user, we can have a confirmation question in the update command.

Reply:
Let's take note of this in consideration for a future, global `--silent` flag.

Reply:
Note that the `system update` command sets precedent with `--skip` flag. Different, but somewhat related in concept.

Reply:
Got it. @leecalcote but how should we handle it now? When the user runs `system update`, it will call the `reset()` function which will ask for confirmation from the user. If the user responds "no", the `system update` command would not consider it and still continue with the update process. This should be changed.

Reply:
Currently, (in the released versions), `mesheryctl system update` does not call `reset()`. I haven't read through this whole issue. Is there a need to change this behavior?

Reply:
It resets first and then proceeds to update doesn't it. We do not need to change this... The issue is that since we added a user confirmation in the reset function, we need to also add handling for it in the update command.